index.css

```diff
@@ -0,0 +1,107 @@
+:root {
+    --primary-color: #002D62;
+    --secondary-color: white;
+    --primary-shade-color:  #194271; 
+    --font-family: sans-serif;
+    font-size: 16px; 

```

Comment:
You can set `font-size: 100%`, that way it'll match the font-size set by users in their browser. 16px is the default of course but user can set it differently according to their preferences. So for example if the user has set font-size to be 10px then setting `font-size: 100%` will automatically be 10px, and so on.

The point here is to use relative units wherever possible.